### PR TITLE
add foreman-proxy

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/foreman-proxy/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+foreman_proxy_image: quay.io/evgeni/foreman-proxy-rpm:3.12
```
Comment (ehelms) [inline roles/foreman-proxy/defaults/main.yaml:2]: Which repository hosts this Containerfile?

Comment (evgeni) [reply roles/foreman-proxy/defaults/main.yaml:2]: https://github.com/evgeni/foreman-container/blob/devel/Containerfile.proxy

File: roles/foreman-proxy/templates/pulpcore.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+:enabled: true
+:pulp_url: http://localhost:8080/
+:content_app_url: http://localhost:8080/pulp/content/
```
[the remark this answers is not in the document]
Comment (evgeni) [reply roles/foreman-proxy/templates/pulpcore.yaml.j2:4]: Yes.
I can make those variables (with default values pointing at localhost), would that help?

The real values would come when we actually integrate pulp

[the remark this answers is not in the document]
Comment (ehelms) [reply roles/foreman-proxy/templates/pulpcore.yaml.j2]: I am mostly just trying to think ahead about keeping an eye on to-dos.

[the remark this answers is not in the document]
Comment (evgeni) [reply roles/foreman-proxy/templates/pulpcore.yaml.j2]: https://github.com/ehelms/foreman-quadlet/issues/13

**Review by ehelms — APPROVED** [review]
Could you either add the Containerfiles to this repo to consolidate or add a link to the README to help finding them? I know @Odilhao was interested in looking over all of them

**Comment by evgeni** [comment]
> Could you either add the Containerfiles to this repo to consolidate or add a link to the README to help finding them? I know @Odilhao was interested in looking over all of them

Readme of the role or the repo?

**Comment by evgeni** [comment]
https://github.com/ehelms/foreman-quadlet/pull/14

**Pull request merged**